src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -2181,12 +2181,26 @@ static void Main()
     static string M2(string x) => throw null!;
 }";
 
-            // missing warnings
-            // Tracked by https://github.com/dotnet/roslyn/issues/32698
```

Comment:
>// Tracked by https://github.com/dotnet/roslyn/issues/32698 [](start = 12, length = 59)

Can  [https://github.com/dotnet/roslyn/issues/32698](https://github.com/dotnet/roslyn/issues/32698) also be resolved?